domains/gioco.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "giocoliere",
+        "email": "simcrigjeki@gmail.com"
+    },
+    "record": {
+        "URL": "giocoliere.github.io/crirolled/"
```

```suggestion
        "CNAME": "giocoliere.github.io"
```

I cannot reach this website, and please use a valid record like the one suggested above.